packages/payload/src/admin/components/forms/field-types/Row/index.tsx

```diff
@@ -34,7 +34,7 @@ const Row: React.FC<Props> = (props) => {
           fieldTypes={fieldTypes}
           forceRender={forceRender}
           indexPath={indexPath}
-          margins={false}
+          margins="small"
```

Comment:
This change is questionable. Need to test this in other places to check for unintended consequences.

Reply:
Reverted now